money/README.md

```diff
@@ -0,0 +1,51 @@
+title: com.iluwatar.Money Design Pattern
+title: Money Design Pattern
+category: Functional
+language: en
+tag:
```

Comment:
Are there any tags we could use from https://github.com/iluwatar/java-design-patterns/wiki/07.-Categories-and-Tags ?